examples/update_flow_control/README.md

```diff
@@ -0,0 +1,109 @@
+# mender-client-python-example
+
+## Overview
+
+This example shows how to control the Update flow of the Mender client using the
+DBus API.
+
+At start time, it sets an Update Control Map that instructs the Mender client to
+"pause" on every possible state. These are `ArtifactInstall_Enter`,
+`ArtifactReboot_Enter`, and `ArtifactCommit_Enter`. Learn more about the Mender
+client states in the [States section from Mender Docs
+page](https://docs.mender.io/overview/state-script#states).
+
+Then the application waits for user input. The user can select a new Update
+Control Map to send to the Mender client. Each of the possible maps set a one of
+the previously mentioned states to "continue", which allows any ongoing update
+to progress.
+
+The code uses the Mender DBus API `io.mender.Update1` to set the Update Control
+Map in the Mender client. Refer to Mender Docs for more details on the actual
+[`io.mender.Update1.SetUpdateControlMap`
+method](https://docs.mender.io/development/device-side-api/io.mender.update1#setupdatecontrolmap)
+being used.
+
+## Usage
+
+See [main README](../../README.md) for instructions on how to install this
+package. Once installed, start this example with:
+
+```
+mender-update-flow-control
+```
+
+Which initializes the application, setting the "pause" all Update Control Map,
+and asks the user for further input with the following prompt:
+
+```
+Current map is:
+{
+    "priority": -1,
+    "states": {
+        "ArtifactInstall_Enter": {
+            "action": "pause"
+        },
+        "ArtifactReboot_Enter": {
+            "action": "pause"
+        },
+        "ArtifactCommit_Enter": {
+            "action": "pause"
+        }
+    },
+    "id": "01234567-89ab-cdef-0123-456789abcdef"
+}
+-----------------------
+What do you want to do?
+  0) Pause on all states
+  1) Continue with Installing new software
```

Comment:
I'm not sure where it fits best, but I would include a mention about `on_action_executed`. What it does is to *atomically* execute the action of the map, and then set the next action, so that you don't risk race conditions when continuing an update and you want to pause the next update.

For example, with:
```
{
    "states": {
        "ArtifactInstall_Enter": {
            "action": "continue",
            "on_action_executed": "pause"
        },
    },
    "id": "01234567-89ab-cdef-0123-456789abcdef"
}
```

you don't need to set a new map to reinstate a pause. This should be our official recommendation.

Reply:
:+1: 

I'll add this in the new section "Going to production" in the documentation, not in the code.